ProductionModePlugin.js

```diff
@@ -67,34 +67,37 @@ class ProductionModePlugin {
         if(param.isString() && param.string === "globalize" && this.moduleFilter(request) &&
           !(globalizeCompilerHelper.isCompiledDataModule(request))) {
 
-          // Statically extract Globalize formatters and parsers from the request
-          // file only. Then, create a custom precompiled formatters/parsers module
-          // that will be called instead of Globalize, which in turn requires
-          // Globalize, set the default locale and then exports the Globalize
-          // object.
-          const compiledDataFilepath = globalizeCompilerHelper.createCompiledDataModule(request);
+          this.supportedLocales.forEach((locale) => {
```

Comment:
Thank you! Very clear now. We should add that clarification as an internal comment into the code :).

> I wish I knew how to create "virtual" modules (non-file backed)…

Me too ☺️ https://stackoverflow.com/questions/31147779/is-there-a-webpack-dependency-object-that-takes-content-instead-of-filepath-de